docs/preview/index.md

```diff
@@ -134,14 +128,14 @@ Instance running!
 Visit https://127.0.0.1
 ```
 ``` bash
-firefox https://localhost
+firefox https://127.0.0.1
 ```
 
 **Notes and Known Issues**
 
 - You may see the following error message `TypeError: Object.fromEntries is not a function`.
   This means you need to update your base Invenio docker image because node 13+ is needed.
-  Run `docker pull inveniosoftware/centos7-python:3.6` before running `invenio-cli containerize --pre` again.
+  Run `docker pull inveniosoftware/centos8-python:3.6` (adapt it to your python version)  before running `invenio-cli containerize --pre` again.
```

Comment:
Python 3.6 is still CentOS 7, Python 3.7 and 3.8 are on top of CentOS 8. This is due to base image python version support.
```suggestion
  Run `docker pull inveniosoftware/centos7-python:3.6` or `docker pull inveniosoftware/centos8-python:3.{7, 8}` (adapt it to your python version)  before running `invenio-cli containerize --pre` again.
```